### ISSUE TITLE
[Documentation] MSAL features

### ISSUE THREAD

**Issue by maliksahil** [AzureAD/microsoft-authentication-library-for-dotnet#2714]
**Documentation related to component**
MSAL

**Please check those that apply**

- [ ] typo
- [ ] documentation doesn't exist
- [x ] documentation needs clarification
- [ ] error(s) in example
- [ ] needs example

**Description of the issue**
From this page, https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/MSAL.NET-supports-multiple-application-architectures-and-multiple-platforms

Specifically this paragraph, "or because of limitations of the platform itself (for instance .NET Core does not provide UI, and therefore acquisition of tokens requiring user interaction through a Web browser is not possible in .NET Core)."

 ..  seems inaccurate. We should perhaps reword this to make it clearer?

**Comment by bgavrilMS** [comment]
Thanks for pointing this out @maliksahil . I fixed it.
